<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CharacterMapper</class>
 <widget class="QWidget" name="CharacterMapper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Character Mapper</string>
  </property>
  <layout class="QVBoxLayout" name="main_layout">
   <item>
    <layout class="QHBoxLayout" name="lists_layout">
     <item>
      <widget class="QGroupBox" name="mapping_group">
       <property name="title">
        <string>Character Bone Mapping</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QListWidget" name="mapping_list"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="assign_layout">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="assign_btn">
         <property name="text">
          <string>&lt;&lt; Assign</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="objects_group">
       <property name="title">
        <string>Scene Objects</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QListWidget" name="objects_list"/>
        </item>
        <item>
         <widget class="QPushButton" name="refresh_btn">
          <property name="text">
           <string>Refresh</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="actions_group">
     <property name="title">
      <string>Actions</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QGridLayout" name="actions_grid">
        <item row="1" column="0">
         <widget class="QPushButton" name="save_btn">
          <property name="text">
           <string>Save Preset</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="clear_btn">
          <property name="text">
           <string>Clear Mapping</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="load_btn">
          <property name="text">
           <string>Load Preset</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="characterize_btn">
          <property name="text">
           <string>Create Character</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="export_btn">
          <property name="text">
           <string>Export Preset...</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="import_btn">
          <property name="text">
           <string>Import Preset...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="preset_layout">
        <item>
         <widget class="QLabel" name="preset_label">
          <property name="text">
           <string>Preset Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="preset_name_edit"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
